DYDX-Bots/Insolvent_Contract/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "dydx-15",
+  "version": "0.0.1",
+  "description": "Detect insolvent contract",
```

```suggestion
  "description": "Detect insolvent contract in dYdX Liquidity Module",
```
a suggestion to improve description